packages/design-system/src/components/SlidingPanel/SlidingPanelToggle.jsx

```diff
@@ -0,0 +1,63 @@
+import Button from '../Button/Button';
+import PropTypes from 'prop-types';
+import React from 'react';
+import classNames from 'classnames';
+
+/**
+ * A link that triggers the visibility of a sliding panel
+ */
+export class SlidingPanelToggle extends React.PureComponent {
+  componentDidUpdate(prevProps) {
+    if (!this.props.panelOpen && prevProps.panelOpen && this.buttonRef) {
+      this.buttonRef.focus();
+    }
+  }
+
+  render() {
+    const { className, children, inline, showPanel, panelOpen, ...others } = this.props;
+    const classes = classNames(
+      'ds-c-sliding-panel__toggle',
+      inline && 'ds-c-sliding-panel__toggle--inline',
+      className
+    );
+
+    return (
+      <Button
+        className={classes}
+        inputRef={(el) => (this.buttonRef = el)}
+        onClick={() => showPanel()}
```

Comment:
[nitpick] I know this is how it was setup in the original panel, but consider updating this to be
```
onClick={showPanel}
```
I think the extra anonymous function is unnecessary here